### PR TITLE
Giving more details on configuration

### PR DESCRIPTION
**Pull request by paigerube14**
Adding more details on different components on the configuration for krkn 

Also noticed requirements issue: 
`
:0: UserWarning: You do not have a working installation of the service_identity module: 'No module named 'service_identity''.  Please install it from <https://pypi.python.org/pypi/service_identity> and make sure all of its dependencies are satisfied.  Without the service_identity module, Twisted can perform only rudimentary TLS client hostname verification.  Many valid certificate/hostname mappings may be rejected
`

### CODE REVIEW

File: docs/config.md [hunk 1]
```diff
@@ -2,3 +2,71 @@
...
+## Publish kraken status
+**publish_kraken_status**: Can be accessed at http://0.0.0.0:8081 (or what signal_address and port you set in signal address section)
+**signal_state**: State you want kraken to start at; will wait for the RUN signal to start running a chaos iteration. When set to PAUSE before running the scenarios, refer to [signal.md](signal.md) for more details
+
+## Signal Address 
+**signal_address**: Address to listen/post the signal state to
+**port**: port to listen/post the signal state to
+
+## Litmus Variables
+Litmus installation specifics if you are running one of the hog scenarios. See [litmus doc](litmus_scenarios.md) for more informaiton on these types of scenarios
```
Comment (chaitanyaenr) [inline docs/config.md:31]: s/informaiton/information

File: docs/config.md [hunk 2]
```diff
@@ -2,3 +2,71 @@
...
+Litmus installation specifics if you are running one of the hog scenarios. See [litmus doc](litmus_scenarios.md) for more informaiton on these types of scenarios 
+**litmus_install**: Installs specified version of litmus, set to False if it's already setup
+**litmus_version**: Litmus version to install
+**litmus_uninstall**: If you want to uninstall litmus if failure
+**litmus_uninstall_before_run**: If you want to uninstall litmus before a new run starts, True or False values
+
+
+## Chaos Scenarios 
+
+**chaos_scenarios**: List of different types of chaos scenarios you want to run with paths to their specific yaml file configurations
```
Comment (chaitanyaenr) [inline docs/config.md:40]: @paigerube14 Maybe mention that the scenarios are currently run one after other and will exit if one of the scenarios fail without moving onto the next one. Thoughts?

**Review by chaitanyaenr — COMMENTED** [review]
Minor nits

**Review by tsebastiani — APPROVED** [review]

**Review by chaitanyaenr — APPROVED** [review]
LGTM!

**Pull request merged**